scripts/zkai_d128_recursive_pcd_route_selector_gate.py

```
@@ -0,0 +1,910 @@
+        "result": RESULT,
+        "issue": ISSUE,
+        "source_evidence": source_evidence,
+        "local_repo_probe": repo_probe,
+        "route_table": routes,
+        "route_decision": route_decision,
+        "non_claims": list(NON_CLAIMS),
+        "validation_commands": list(VALIDATION_COMMANDS),
+    }
+
+
+def validate_source_evidence(value: Any) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)